### ISSUE TITLE
CGPoint.interpolate EXC_BREAKPOINT 

### ISSUE THREAD

**Issue by rk85097** [airbnb/lottie-ios#1064]
After long time of not upgrading RN and Lottie, I did it, and suddenly ONLY on release mode in iOS, I get many crashes with the same exception. Crash report attached. On debug mode - all working as expected.

## Check these before submitting:
- [x] The issue doesn't involve an [Unsupported Feature](https://github.com/airbnb/lottie-ios/blob/master/README.md#supported-after-effects-features)
- [v] This issue isn't related to another open issue (https://github.com/airbnb/lottie-ios/issues/934)

## This issue is a:
- [] Non-Crashing Bug (Visual or otherwise)
- [x] Crashing Bug
- [] Feature Request
- [x] Regression (Something that once worked, but doesn't work anymore)

## Which Version of Lottie are you using?

 - lottie-ios (3.1.3)
 - lottie-react-native (3.2.1):

## What Platform are you on? 

- [] MacOS
- [x] iOS

## What Language are you in?
- [x] Swift
- [] Objective-C

## Animation JSON
It is impossible to understand which JSON is possibly the problematic one as we run many of them. I could not locate the problematic one - keep in mind that all of the JSONs are working as expected on debug mode. 

## Crash Report
[LINK](https://sentry.io/share/issue/8756d5ebba074721a5f490b8e78042b1/)

**Comment by rk85097** [comment]
Update: removing this part of code in native ios fix it, but I'm not sure why and how it will affect other things...

```
      if accurateDistance < point.distance {
        closestPoint = closestPoint - 1
        point = points[closestPoint]
        pointAmount = CGFloat(closestPoint) * step
        nextPointAmount = pointAmount + step
        if closestPoint < 0 {
          foundPoint = true
        }
        continue
      }
```

**Comment by levantAJ** [comment]
I got the same issue and 100% on iOS 10

**Crashed: com.apple.main-thread
EXC_BREAKPOINT 0x000000000000defe**

**lottie-ios 3.1.2**


```
Crashed: com.apple.main-thread
0  Lottie                         0x1d9b798 specialized CGPoint.interpolate(_:outTangent:inTangent:amount:maxIterations:samples:accuracy:) (<compiler-generated>)
1  Lottie                         0x1d9b655 specialized CGPoint.interpolate(_:outTangent:inTangent:amount:maxIterations:samples:accuracy:) (<compiler-generated>)
2  Lottie                         0x1d85f3f specialized Vector3D.interpolateTo(_:amount:spatialOutTangent:spatialInTangent:) (<compiler-generated>)
3  Lottie                         0x1d85807 protocol witness for Interpolatable.interpolateTo(_:amount:spatialOutTangent:spatialInTangent:) in conformance Vector3D (<compiler-generated>)
4  Lottie                         0x1d8c6b1 Keyframe.interpolate(_:progress:) + 40 (KeyframeExtensions.swift:40)
5  Lottie                         0x1d8ee0f KeyframeInterpolator.value(frame:) + 83 (KeyframeInterpolator.swift:83)
6  Lottie                         0x1d90987 protocol witness for AnyValueProvider.value(frame:) in conformance KeyframeInterpolator<A> (<compiler-generated>)
7  Lottie                         0x1d9c95f NodeProperty.update(frame:) + 43 (NodeProperty.swift:43)
8  Lottie                         0x1d9c9ed protocol witness for AnyNodeProperty.update(frame:) in conformance NodeProperty<A> (<compiler-generated>)
9  Lottie                         0x1d9cd9d specialized NodePropertyMap.updateNodeProperties(frame:) (<compiler-generated>)
10 Lottie                         0x1d642a1 specialized AnimatorNode.updateContents(_:forceLocalUpdate:) + 145 (AnimatorNode.swift:145)
11 Lottie                         0x1d6ec4f CompositionLayer.displayWithFrame(frame:forceUpdates:) + 157 (AnimatorNode.swift:157)
12 Lottie                         0x1da1709 PreCompositionLayer.displayContentsWithFrame(frame:forceUpdates:) + 95 (PreCompositionLayer.swift:95)
13 Lottie                         0x1d6ed2f CompositionLayer.displayWithFrame(frame:forceUpdates:) + 114 (CompositionLayer.swift:114)
14 Lottie                         0x1d572ef AnimationContainer.display() + 176 (AnimationContainer.swift:176)
15 Lottie                         0x1d5734f @objc AnimationContainer.display() (<compiler-generated>)
16 QuartzCore                     0x1e319b9f CA::Layer::display_if_needed(CA::Transaction*) + 208
17 QuartzCore                     0x1e319c0d CA::Layer::layout_and_display_if_needed(CA::Transaction*) + 24
18 QuartzCore                     0x1e2a8839 CA::Context::commit_transaction(CA::Transaction*) + 320
19 QuartzCore                     0x1e2c6fdb CA::Transaction::commit() + 578
20 QuartzCore                     0x1e2c7b2f CA::Transaction::observer_callback(__CFRunLoopObserver*, unsigned long, void*) + 162
21 CoreFoundation                 0x1b45f803 __CFRUNLOOP_IS_CALLING_OUT_TO_AN_OBSERVER_CALLBACK_FUNCTION__ + 20
22 CoreFoundation                 0x1b45da55 __CFRunLoopDoObservers + 282
23 CoreFoundation                 0x1b45e017 __CFRunLoopRun + 1358
24 CoreFoundation                 0x1b3b11af CFRunLoopRunSpecific + 470
25 CoreFoundation                 0x1b3b0fd1 CFRunLoopRunInMode + 104
26 GraphicsServices               0x1cb5bb41 GSEventRunModal + 80
27 UIKit                          0x20733a53 UIApplicationMain + 150
28 MyApp                       0xe84ed main + 30 (MyClass.swift:30)
29 libdyld.dylib                  0x1ab9e4eb start + 2
```

Could anyone help, thanks?

**Comment by buba447** [comment]
Should be fixed in 3.1.5 release! Cheers
